LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-handler-throws-expected.txt

```diff
@@ -1,3 +1,6 @@
+CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: a message
+
+Harness Error (FAIL), message = Unhandled rejection: a message
```

Comment:
This happens with quite some tests and is kind of unfortunate. For this kind of test where the handler just throws an exception we did not find a good solution. Having said that, I suppose it would still count as pass on wpt.fyi test score.